src/components/registration/steps.jsx

```diff
@@ -473,8 +489,21 @@ module.exports = {
         onChooseOrganization: function (name, values) {
             this.setState({otherDisabled: values.indexOf(this.organizationL10nStems.indexOf('orgChoiceOther')) === -1});
         },
-        onValidSubmit: function (formData, reset, invalidate) {
+        validateOrganizationType: function (formData) {
```

Comment:
If this were a regular validation for the organization CheckboxGroup, then wouldn't it automatically be run `onSubmit` (and ahead of `onValidSubmit`)?